src/components/Editor/Properties/PropertyColor.vue

```diff
@@ -104,25 +103,12 @@ export default {
 	methods: {
 		/**
 		 * Changes / Sets the custom color of this event
-		 *
-		 * The problem we are facing here is that the
-		 * color-picker component uses normal hex colors,
-		 * but the RFC 7986 property COLOR requires
-		 * css-color-names.
-		 *
-		 * The color-space of css-color-names is smaller
```

Comment:
We don't need debounce because submit happens only once?

Reply:
Yep and it doesn't happen immediately after choosing the color

Reply:
Nice!